changelogs/fragments/migrate_iam_group.yml

```diff
@@ -0,0 +1,4 @@
+major_changes:
+- iam_group - The module has been migrated from the ``community.aws`` collection.
+  Playbooks using the Fully Qualified Collection Name for this module should be updated
+  to use ``community.aws.iam_group``.
```

Comment:
Shouldn't this be `amazon.aws.iam_group`?

Reply:
fixed